src/common/TokenParser.ts

```diff
@@ -183,7 +197,6 @@ export class TokenParser {
             promises.push(ERC20Contract.findById(tokenContract).then((token: any) => {
                 return {
                     address: token.address,
-                    decimals: token.decimals
```

Comment:
Why remove `decimals`?

Reply:
It was not mentioned in original task, but added in next PR as well as `symbol`, `name`